src/packagedef.jl

```diff
@@ -1,13 +1,40 @@
 @eval Base.Experimental.@optlevel 1
 
-using FileWatching, REPL, Distributed, UUIDs
+using FileWatching, REPL, UUIDs
 import LibGit2
 using Base: PkgId
 using Base.Meta: isexpr
 using Core: CodeInfo
 
 export revise, includet, entr, MethodSummary
 
+
+# Wrapper struct to indicate a worker belonging to the Distributed stdlib. Other
+# libraries should make their own types for Revise to dispatch on.
+struct DistributedWorker
```

Comment:
Does this have to be here or could it be in the extension? I guess it's here because of `init_worker(p::Int)` below?

Also, should we define an `AbstractWorker` and then suggest people subtype? I'm not sure it helps concretely, but it seems mildly tidier to me.

Reply:
Yeah exactly it's because of the `init_worker(::Int)` definition. Technically I think that could also go in the extension, but I slightly lean towards keeping it in the main package since the extension doesn't really 'own' the `Int` type that we're dispatching on.

Agreed about the `AbstractWorker` type, added it in b468b35.